modules/apps/user/user-test/src/testIntegration/java/com/liferay/user/model/impl/test/UserImplTest.java

```diff
@@ -51,6 +52,13 @@ public void setUp() throws Exception {
 		_user = UserTestUtil.addUser(RandomTestUtil.randomString());
 	}
 
+	@Test
+	public void testFetchContact() {
+		Contact contact = _user.fetchContact();
+
+		Assert.assertEquals(contact.getContactId(), _user.getContactId());
+	}
+
```

Comment:
Also the test is quite strange: I mean... your change is about checking null values... this test checks what? 😄

Reply:
I could definitely use your input on this.

My thought process was:

1. In the setup, the user is already randomly created with a random contact.
2. Previously, before my PR, if the test were to be run, we would always get NULL returned no matter what.
3. With my PR, we will always get the contact that is associated with the user even for "first time" calls which is what happens in the test.